### DISCUSSION TITLE
In project Beta have a default field for issue number

### DISCUSSION THREAD

**Discussion by lahabana** [community/community#14935] [Projects and Issues]
Currently I can have columns for repository but not for issue number:

<img width="224" alt="image" src="https://user-images.githubusercontent.com/702355/163363337-70e5a2c4-c3cf-42fe-9dca-8966bf97de73.png">

Potentially a dupe of: https://github.com/github/feedback/discussions/12879 or https://github.com/github/feedback/discussions/5904 but they don't explicitly state as having it a field.

**Comment by lucas-labs** [comment]
This would be great. Having the ability to filter by issue number and easily search for a specific issue is a must in my opinion.
